src/sentry/api/endpoints/project_profiling_profile.py

```diff
@@ -133,6 +133,19 @@ def get(self, request: Request, project: Project, profile_id: str) -> HttpRespon
         return proxy_profiling_service(**kwargs)
 
 
+@region_silo_endpoint
+class ProjectProfilingFlamegraphEndpoint(ProjectProfilingBaseEndpoint):
+    def get(self, request: Request, project: Project, profile_id: str) -> HttpResponse:
+        if not features.has("organizations:profiling", project.organization, actor=request.user):
+            return Response(status=404)
+        kwargs: Dict[str, Any] = {
+            "method": "GET",
+            "path": f"/organizations/{project.organization.id}/projects/{project.id}/flamegraph",
```

Comment:
```suggestion
            "path": f"/organizations/{project.organization_id}/projects/{project.id}/flamegraph",
```

Using `project.organization` will incur another database query.

Reply:
@markstory good point.

I'm going to fix the other 4 endpoints too 👍🏻

Reply:
The base `ProjectsEndpoint` prefetches the organization when selecting the project (see [here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/project.py#L104)) so it should not incur another database query in this case but this is a good point to be aware of 👍